src/components/form/Editor.js

```diff
@@ -403,10 +403,13 @@ class Editor extends Component {
             if (typeof dataIndices[CommonConstants.GT_ROW_ID] !== CommonConstants.UNDEFINED) {
                 delete dataIndices[CommonConstants.GT_ROW_ID];
             }
-            fetch(ajaxUrl, {
-                method: settings.method,
-                body: JSON.stringify(dataIndices)
-            }).then(response => response.json()).then((data) => {
+            (settings.method == 'PROMISE'?
```

Comment:
This must be strict comparison no reason to use type-converting and the constant should be in CommonConstants.js file as others - you can see this all around the code.